src/modules/esl-share/buttons/pusha.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable max-len */
```

Comment:
```suggestion
```
What was the reason?

Reply:
The lines of button configs are extra long. They exceed 160 symbols.